cmd/check-reaper/main.go

```diff
@@ -135,7 +135,7 @@ func deleteFilteredCheckerPods(client *kubernetes.Clientset, reapCheckerPods map
 			}
 
 			// Delete if there are more than 5 checker pods with the same name in status Succeeded that were created more recently
-			if successOldCount > MaxPodsThreshold && successCount > MaxPodsThreshold {
+			if  v.Status.Phase == "Succeeded" && successOldCount > MaxPodsThreshold && successCount > MaxPodsThreshold {
```

Comment:
Instead of hard-coding `"Succeeded"` I think you should use the const from the k8s [corev1 pkg](https://godoc.org/k8s.io/api/core/v1#PodPhase)

Reply:
Same with line `150` below